assets/css/main.css

```diff
@@ -0,0 +1,159 @@
+:root {
+  --header-height: 60px;
+  --footer-height: 80px;
+}
+
+/* START: Moodle theme styling ported for consistency */
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  font-size: 0.9375rem;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 0;
+}
+
+h1 {
+  font-size: 2.34375rem;
+}
+
+h2 {
+  font-size: 1.875rem;
+}
+
+h3 {
+  font-size: 1.640625rem;
+}
+
+h4 {
+  font-size: 1.40625rem;
+}
+
+h5 {
+  font-size: 1.171875rem;
+}
+
+h6 {
+  font-size: 0.9375rem;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  margin-bottom: .5rem;
+  margin-top: 0;
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+a {
+  color: #0f6cbf;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+a:hover {
+  color: #094478;
+  text-decoration: underline;
+}
+
+blockquote {
+  margin: 0 0.5rem 1rem;
+  padding-left: 1rem;
+  color: #495057;
+  border-left: 5px solid #ced4da; }
+
+/* END: Moodle theme styling ported for consistency */
```

Comment:
@MReyna12 , @P-Gill97 - I want to actively point out that I did this for the time being :joy: . It was mostly to help me compare things visually with the Moodle version while sanity checking as I found it useful. I'm sure we will revisit some of this in subsequent issues.